hive_metastore_client/hive_metastore_client.py

```diff
@@ -212,3 +212,29 @@ def _validate_lists_length(list_a: List[Any], list_b: List[Any]) -> None:
                 "m=_validate_lists_length, msg=The length of the two provided "
                 "lists does not match"
             )
+
+    def get_partition_keys(self, db_name: str, table_name: str) -> List[FieldSchema]:
+        """
+        Gets the partition keys from a table.
```

Comment:
by looking at `get_partition_keys` & `get_partition_keys_names` it's a little hard to get the difference between these methods.

I'm wondering if you can emphasize that this method returns the FieldSchema object (even with the returning type defined in the method signature :thinking: )